provider-blackduck/src/main/java/com/synopsys/integration/alert/provider/blackduck/action/BlackDuckGlobalFieldModelTestAction.java

```diff
@@ -77,18 +77,34 @@ public MessageResult testConfig(String configId, FieldModel fieldModel, FieldUti
             Exception errorException = connectionResult.getException().orElse(null);
             if (RestConstants.UNAUTHORIZED_401 == connectionResult.getHttpStatusCode()) {
                 throw AlertFieldException
-                          .singleFieldError(String.format("Invalid credential(s) for: %s. %s", url, failureMessage), BlackDuckDescriptor.KEY_BLACKDUCK_API_KEY, "This API Key isn't valid, try a different one.");
+                    .singleFieldError(
+                        String.format("Invalid credential(s) for: %s. %s", url, failureMessage),
+                        BlackDuckDescriptor.KEY_BLACKDUCK_API_KEY,
+                        "This API Key isn't valid, try a different one."
+                    );
             } else if (connectionResult.getHttpStatusCode() > 0) {
                 // TODO why are we throwing a non-alert exception?
                 HttpUrl connectionUrl = new HttpUrl(url);
-                throw new IntegrationRestException(HttpMethod.GET, connectionUrl, connectionResult.getHttpStatusCode(), String.format("Could not connect to: %s", url), failureMessage, errorException);
+                throw new IntegrationRestException(
+                    HttpMethod.GET,
+                    connectionUrl,
+                    connectionResult.getHttpStatusCode(),
+                    String.format("Could not connect to: %s", url),
+                    failureMessage,
+                    errorException
+                );
             }
             throw new AlertException(String.format("Could not connect to: %s. %s", url, failureMessage), errorException);
         }
 
         BlackDuckApiTokenValidator blackDuckAPITokenValidator = new BlackDuckApiTokenValidator(blackDuckProperties);
         if (!blackDuckAPITokenValidator.isApiTokenValid()) {
-            throw AlertFieldException.singleFieldError(BlackDuckDescriptor.KEY_BLACKDUCK_API_KEY, "User permission failed. Cannot read notifications from Black Duck.");
+            throw AlertFieldException
+                .singleFieldError(
+                    "User permission failed. Cannot read notifications from Black Duck.",
+                    BlackDuckDescriptor.KEY_BLACKDUCK_API_KEY,
+                    "User permission failed. Cannot read notifications from Black Duck."
+                );
```

Comment:
How come the change in which constructor to use?
Why is the message and fieldError the same thing?

Reply:
This was an earlier oversight that was causing no message to be created when an  exception was thrown. We would attempt to log the message in the response model but since no message was created we would see a "null" in  the message body in the Alert logs.

In this case, the message and the field error would be the same so I just duplicated it in both places. Functionally,  there should be no difference, this is just to clean up some ugly "null" messages in the log.